roles/base/templates/run-ansible-pull.sh.j2

```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
```

Comment:
On 2016-01-16 12:10 -0800, Sean Kelly wrote:
> In roles/base/templates/run-ansible-pull.sh.j2:
> 
> > @@ -1,4 +1,4 @@
> > -#! /bin/sh
> > +#! /bin/bash
> 
> Bash is at /usr/local/bin on FreeBSD.

Is it really a bash script?


Amar.